admin/database/postgres/migrations/0031.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX assets_created_on_idx ON assets (created_on);
```

Comment:
Of course always hard to say, but I don't think this index will be useful – indexes mostly only help if they significantly reduce the number of rows to scan. In this case:
1. Looking for assets not present in `projects` will likely lead to a full table scan anyway
2. Most assets will have been created before the `createdBefore`, so doesn't reduce rows scanned significantly

Reply:
Good point.
1. Don't think it is true. When I did an explain on the query(`left join with id = null`), it was filtering the rows first and then joining the tables. 
2. Yes makes sense.